src/bindings/python/src/compatibility/ngraph/opset8/ops.py

```diff
@@ -196,19 +178,19 @@ def multiclass_nms(
 
 @nameable_op
 def matrix_nms(
-    boxes: NodeInput,
-    scores: NodeInput,
-    sort_result_type: str = "none",
-    sort_result_across_batch: bool = False,
-    output_type: str = "i64",
-    score_threshold: float = 0.0,
-    nms_top_k: int = -1,
-    keep_top_k: int = -1,
-    background_class: int = -1,
-    decay_function: str = "linear",
-    gaussian_sigma: float = 2.0,
-    post_threshold: float = 0.0,
-    normalized: bool = True
+        boxes: NodeInput,
```

Comment:
Why there are new dilatations added all around?

Reply:
It is fine and now it gets aligned with other function definitions. I applied PyCharm automatic formatting